### PR TITLE
Match spec for referral request creation and voiding

### PR DESCRIPTION
**Pull request by gigxz**
https://docs.google.com/spreadsheets/d/1Agy0ama7gerOyYDuZclKb_p0hUXV359w/edit#gid=1629128671 plus requestorName which wasn't documented there

### CODE REVIEW

**Review by ttoomey — APPROVED** [review]
Looks great! Good to know we are expecting camelCase params

**Comment by gigxz** [comment]
Yeah there is an example payload in the spreadsheet!

**Comment by gigxz** [comment]
I'll let you merge this in when you want @ttoomey

**Pull request merged**
